symbols/yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```

Comment:
Can you please check this file needs to commit or not?

Reply:
Sure. @emclaug2 @daileytj

Reply:
It looks like you run `yarn` by accident. This repository has a `package.json` file, but its dependencies list is empty; a `yarn.lock` file is not needed in the `symbols` folder.